ios/RNBranch.m

```diff
@@ -167,10 +138,10 @@ + (void)initSessionWithLaunchOptions:(NSDictionary *)launchOptions isReferrable:
     savedLaunchOptions = launchOptions;
     savedIsReferrable = isReferrable;
 
-    [self.branch registerPluginName:@"ReactNative" version:RNBNC_PLUGIN_VERSION];
+    [self.branch registerPluginName:@"ReactNative" version:RNBRANCH_VERSION];
 
     // Can't currently support this on Android.
-    // if (!deferInitializationForJSLoad && !RNBranchConfig.instance.deferInitializationForJSLoad) [self initializeBranchSDK];
+    // if (!deferInitializationForJSLoad && !BranchJsonConfig.instance.deferInitializationForJSLoad) [self initializeBranchSDK];
```

Comment:
Didn't remove these references so we can have a starting point in case this gets implemented in native SDKs